### PR TITLE
New tools icon

### PR DESCRIPTION
**Pull request by spencerdearman**
Merging in the updated tools icon

### CODE REVIEW

File: DiveMeets/DiveMeets/Views/Main/FloatingMenuBar.swift [hunk 1]
```diff
@@ -56,18 +56,24 @@ struct FloatingMenuBar: View {
         "gearshape.circle": 1.6,
         "person.circle": 1.6,
         "magnifyingglass.circle": 1.5,
+        "wrench.and.screwdriver": 1.2,
```
Comment (lsherwin10) [inline DiveMeets/DiveMeets/Views/Main/FloatingMenuBar.swift]: How does the scaling look on the menu bar? These numbers were just hardcoded to make them all look the same size

File: DiveMeets/DiveMeets/Views/Main/FloatingMenuBar.swift [hunk 2]
```diff
@@ -163,9 +169,17 @@ struct FloatingMenuBar: View {
                     HStack(spacing: 0) {
                         ForEach(visibleTabs, id: \.rawValue) { tab in
                             Spacer()
-                            Image(systemName: selectedTab == tab
-                                  ? selectedTabImage
-                                  : tab.rawValue)
+                            VStack{
```
Comment (lsherwin10) [inline DiveMeets/DiveMeets/Views/Main/FloatingMenuBar.swift]: Does this need to be in a VStack? I think you can just have the if statement, it looks like it's a VStack of one element

File: DiveMeets/DiveMeets.xcodeproj/project.pbxproj [hunk 3]
```diff
@@ -561,7 +561,7 @@
 				CODE_SIGN_STYLE = Automatic;
 				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_ASSET_PATHS = "\"DiveMeets/Preview Content\"";
-				DEVELOPMENT_TEAM = UCV4CFXWW3;
+				DEVELOPMENT_TEAM = WY8V5FPBV9;
```
Comment (lsherwin10) [inline DiveMeets/DiveMeets.xcodeproj/project.pbxproj:564]: Is this the bundle identifier thing? Mine was also giving me issues, not sure which is correct. Which one is showing up on yours?

File: DiveMeets/DiveMeets/Views/Main/FloatingMenuBar.swift (hunk 1 above)
Comment (spencerdearman) [reply DiveMeets/DiveMeets/Views/Main/FloatingMenuBar.swift]: Made the change

File: DiveMeets/DiveMeets.xcodeproj/project.pbxproj (hunk 3 above)
Comment (spencerdearman) [reply DiveMeets/DiveMeets.xcodeproj/project.pbxproj:564]: WY8V5FPBV9;

**Comment by spencerdearman** [comment]
Made changes, merging

**Pull request merged**